doc/source/whatsnew/v0.20.2.txt

```diff
@@ -42,6 +42,7 @@ Conversion
 ^^^^^^^^^^
 
 - Bug in ``pd.to_numeric()`` in which empty data inputs were causing Python to crash (:issue:`16302`)
+- Bug when broadcasting dataframe to series silencing numpy warning messages in common use case (:issue:`16378`)
```

Comment:
Silence numpy warnings when broadcasting DataFrame to Series with comparison ops.